frontend/app/component/form/polling_station/PollingStationForm.test.tsx

```diff
@@ -7,29 +7,48 @@ import { ErrorResponse, PollingStation } from "@kiesraad/api";
 
 import { PollingStationForm } from "./PollingStationForm";
 
+function getInputs() {
+  const result = {
+    number: screen.getByRole("textbox", { name: "Nummer" }),
+    name: screen.getByRole("textbox", { name: "Naam" }),
+    numberOfVoters: screen.getByRole("textbox", { name: "Aantal kiesgerechtigden Optioneel" }),
+    street: screen.getByRole("textbox", { name: "Straatnaam" }),
+    houseNumber: screen.getByRole("textbox", { name: "Huisnummer" }),
+    houseNumberAddition: screen.getByRole("textbox", { name: "Toevoeging" }),
+    postalCode: screen.getByRole("textbox", { name: "Postcode" }),
+    locality: screen.getByRole("textbox", { name: "Plaats" }),
+    typeOptionFixedLocation: screen.getByRole("radio", { name: "Vaste locatie" }),
+    typeOptionSpecial: screen.getByRole("radio", { name: "Bijzonder" }),
+    typeOptionMobile: screen.getByRole("radio", { name: "Mobiel" }),
+  };
+
+  return result;
+}
+
 async function fillForm(user: UserEvent, testPollingStation: PollingStation | Omit<PollingStation, "id">) {
```

Comment:
This function clicks the "Opslaan" button. Both the test "PollingStationForm create" and "Validation backend errors" click that button in the test as well, while I don't think they should.

Reply:
`fillFormAndSave()` might be a better name in that sense.